functions/definition/models.yaml

```diff
@@ -113,6 +113,15 @@ FbOrganisation:
       # NOTE: We can remove this
       nameKey:
         type: string
+      isArchived:
```

Comment:
Let's add a NOTE that this was added.

Reply:
From the conversation with @thenav56, let's only add isArchived.
It can be optional.